{{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml

```diff
@@ -80,6 +80,9 @@ stages:
 {%- if cookiecutter.environment_strategy == 'dedicated' %}
   - oc tag "${SOURCE}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
            "${TARGET}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
+  - seiso images history "${TARGET}/${CI_PROJECT_NAME}" --force
+{%- else %}
+  - seiso images history "${TARGET}/${CI_PROJECT_NAME}-${CI_ENVIRONMENT_NAME}" --force
```

Comment:
Unfortunately, the deployment of the demo project failed with this change.

It looks like the image stream _never_ has an environment postfix.

I'll prepare a PR to fix this.

Reply:
See #109.